en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -285,20 +285,35 @@ You can also create list data from associations that can be reached with joins::
         'valueField' => 'author.name'
     ])->contain(['Authors']);
 
-Lastly it is possible to use closures to access entity mutator methods in your
-list finds. This example shows using the ``_getFullName()`` mutator method from
+Customize Key-Value Output
+--------------------------
+
+Lastly it is possible to use closures to access entity accessor methods in your
+list finds. ::
+
+    // In your Authors Entity create a virtual field to be used as the displayField:
+    function _getLabel()
+    {
+        return __('Name %s', $this->_properties['name'])
+          . ' / ' . __('User ID %s', $this->_properties['category_id']);
```

Comment:
User ID with category_I'd feels strange.
I don'the really see the advantage of this example...I just feel it is likely to confuse people